### ISSUE TITLE
type: "input" support?

### ISSUE THREAD

**Issue by bahmdev** [lipis/bootstrap-sweetalert#51]
```
`swal({   
title: "An input!",   
text: "Write something interesting:",   
type: "input",   
showCancelButton: true,  
closeOnConfirm: false,   
inputPlaceholder: "Write something" }, 
function(inputValue){   
if (inputValue === false) 
return false;      
if (inputValue === "") {    
swal.showInputError("You need to write something!");     return false   }      
swal("Nice!", "You wrote: " + inputValue, "success"); });
`
```

Is it just not supported or is there an underlying issue?

**Comment by lipis** [comment]
I really have no idea :)

**Comment by alekitto** [comment]
Fixed with cfe1de8
